packages/core/src/libraries/protected-app.ts

```diff
@@ -10,6 +18,17 @@ import {
 export type ProtectedAppLibrary = ReturnType<typeof createProtectedAppLibrary>;
 
 const getProviderConfig = async () => {
+  // Use constant value to skip for integration test, we don't do third party call in integration test
+  if (EnvSet.values.isIntegrationTest) {
+    return {
+      domain: 'protected.app',
+      apiToken: '',
+      accountIdentifier: 'fake_account_id',
+      namespaceIdentifier: 'fake_namespace_id',
+      keyName: 'fake_key_name',
+    };
+  }
```

Comment:
Better not hard code mock data here.

Reply:
Updated, removed this, and use an empty string for "domain" instead.